Ouijesuisraciste

Jokray2 opened the pull request:
Suppression ligne test n°1

Jokray2 committed:
Add files via upload

Jokray2 committed:
Merge pull request #7 from DrTableBasse/master

test

Jokray2 committed:
Add files via upload

Jokray2 committed:
Delete ligneCompleteEtCalculPoints.cpp

Jokray2 committed:
Update ligneCompleteEtCalculPoints.cpp

Jokray2 committed:
Update ligneCompleteEtCalculPoints.cpp

Jokray2 committed:
Merge pull request #10 from DrTableBasse/master

maj

Jokray2 committed:
Update ligneCompleteEtCalculPoints.cpp

Jokray2 committed:
ligneCompleteEtCalculPoints.cpp

Jokray2 committed:
Merge pull request #22 from DrTableBasse/master

test

Jokray2 committed:
Lienn Main.cpp

Ena-Shepherd reviewed (changes requested):
Seems fine except some minor details

Ena-Shepherd commented on src/ligneCompleteEtCalculPoints.cpp:
```diff
@@ -0,0 +1,95 @@
+        if(i==a-1){ //Si on arrive en haut du tableau :
+            for(j=Tab[i];j<20-a;j++){
+                for(k=0;k<10;k++){
+                    board[i][j] = board[j+1+b][k];
```
Variable board : either is a private variable or comes from a specific object.
Is out of scope anyway right now in your code.

Commit:
Refonte du code pour une meilleure gestion de la mémoire et une meilleure lisibilité

Refonte des fichiers 'ligneCompleteEtCalculPoints.cpp', 'board.cpp' et 'main.cpp' pour améliorer la gestion de la mémoire.
Modification de l'allocation dynamique dans la classe Board et modification du fichier board.tab dans main.cpp afin d'améliorer l'utilisation et la gestion de la mémoire.
l'utilisation et la gestion de la mémoire. Modification des fonctions pour une meilleure lisibilité et cohérence du code. J'ai également effectué
petites modifications dans 'difficulty.h' et 'board.h' pour améliorer la lisibilité et l'initialisation des objets.

Commit:
Correction de l'opérateur de comparaison dans la validation du tableau

L'opérateur de comparaison dans le fichier "ligneCompleteEtCalculPoints.cpp" a été correctement mis à jour pour comparer avec un entier (0) au lieu d'une chaîne ('0'). Cela était nécessaire car les entrées board.tab[i][j] sont des valeurs entières. Cette modification permet de vérifier avec précision si une ligne du tableau est complète ou non.

Commit:
"Refactor includes in ligneCompleteEtCalculPoints.cpp

Removed unnecessary includes and added relevant 'board.h' from local directory. Cleaned up for better readability and performance."
